### DISCUSSION TITLE
Log in to account<title>

### DISCUSSION THREAD

**Discussion by kenny0072** [community/community#118880] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am not receiving emails from github containing the code to log in, and I have been trying to resend the code and the code is not being sent to my email address hence I cannot log in

**Comment by ettaboyle** [comment]
The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
